app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,8 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+
+    <color name="actionButton">#286DE6</color>
+    <color name="numberButton">#0C2A5F</color>
```

Comment:
Dat ten color theo link nay nhe
https://chir.ag/projects/name-that-color/#3B4454

Reply:
Mình đăt theo link đấy mà :(((

Reply:
đâu 2 cái button bạn mới thêm kia kìa

Reply:
mình có thêm button nào đâu nhỉ vẫn chỉ có actionButton vs numberButton thôi mà

Reply:
thì đó là màu 2 color đúng ko , 1 màu là royal blue mà sao lại đặt là actionButton

Reply:
à mình hiểu rồi mình tưởng mã màu đặt giống link kia còn tên đặt tn cx đc , h mới hiểu :)))